### PR TITLE
Front on forms, user#show, bicycle#show

### PR DESCRIPTION
**Pull request by cypradoux**
#Navbar (_navbar): 
- fix navbar on top of page
- work on responsivity and hamburger menu
- translate in french
image

#View user show (_user & show)
- avatar profile rounded "replace link by user.avatar"
- users informations
- translate in french informations
- two columns : offered bikes & rentals history
- button delete & home at bottom

image

#View bicycle show (bicycles _bicycle)
- bicycle image
- bicycle informations
- owner email
- price
- add condition to avoid conflict between card & show bicycle

image

#New bicycle/ Sign in/ Log in/ Edit user/ New password  forms (bicycle/_form, registrations/new, sessions/new, registrations/edit, passwords/new)
- center all input
- translate in french
- put style form-control & form-select
image

#Theme_boostrap
- edit on body

#shared/link
-translate in french

### CODE REVIEW

File: app/views/shared/_navbar.html.erb
Comment (JerkStanton) [inline app/views/shared/_navbar.html.erb:1]: That's an idea.

File: app/views/users/show.html.erb
Comment (JerkStanton) [inline app/views/users/show.html.erb:1]: I'll need to merge that.

**Review by JerkStanton — APPROVED** [review]
I saw some interesting things. Espacially the error messages. Can't wait to see it running.

**Review by ikramiste — APPROVED** [review]
No conflicts on my end - OK for merge.  👍

**Pull request merged**